packages/astro/src/core/render/dev/environment.ts

```diff
@@ -32,7 +32,6 @@ export function createDevelopmentEnvironment(
 		site: settings.config.site,
 		ssr: isServerLikeOutput(settings.config),
 		streaming: true,
-		telemetry: Boolean(settings.forceDisableTelemetry),
```

Comment:
Note: There's a bug here before. This should be `telemetry: !settings.forceDisableTelemetry`. Luckily we aren't recording any telemetry in the tests now that affects this.